backend/howtheyvote/pipelines/rcv_list.py

```diff
@@ -84,7 +84,7 @@ def _run(self) -> None:
         # Send Pushover notification
         send_notification(
             title=f"Scraped RCV list for {self.date.strftime('%a, %b %-d')}",
-            message=f"{len(self._vote_ids)} votes, {len(self._vote_group_ids)} vote groups",
+            message=f"{len(self._vote_ids)} votes",
```

Comment:
> a more useful alternative would be to include the number of main votes (which is what would be listed on our website)

Quite often the number in the notification matched that exactly. Not every time, but often enough that I found it to be quite useful. But I am also in favor of making this explicit (and correct in all cases) this is just for informational purposes.